testutil/client.go

```diff
@@ -320,6 +327,7 @@ func HttpLogin(params *LoginParams) (*HttpToken, error) {
 		}
 		return nil, errors.Wrapf(err, "unable to unmarshal the output to get JWTs")
 	}
+	fmt.Println("HttpLogin: outputJson", outputJson)
```

Comment:
@matthewmcneely leaving notes for myself, this needs no action from your end.

cc @joshua-goldstein

note: need to rebase